ajax/libs/ng-bs-daterangepicker/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ng-bs-daterangepicker",
+  "filename": "ng-bs-daterangepicker.min.js",
+  "description": "AngularJS directive for bootstrap-daterangepicker",
+  "version": "0.0.6",
+  "author": {
+    "name": "Luis Farzati",
+    "email": "lfarzati@gmail.com",
+    "url": "http://github.com/luisfarzati"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/luisfarzati/ng-bs-daterangepicker.git"
+  },
+  "keywords": [
+    "angular",
+    "bootstrap",
+    "daterangepicker",
+    "datepicker",
+    "moment",
+    "time",
+    "interval",
+    "calendar"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/luisfarzati/ng-bs-daterangepicker.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "ng-bs-daterangepicker*.!(html)"
```

Comment:
@PeterDaveHello Should we exclude `html` file here? There is no `html` file in recent releases.